src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
+#include <boost/algorithm/string.hpp>
+#include <sstream>
 
 namespace ripple {
 
+class Definitions
+{
+private:
+    std::string
+    translate(std::string inp)
```

Comment:
Pass the param by `const&`, unless I'm missing the reason it's passed by value.

Reply:
I get a lot of these warnings when I do that:
```
warning: returning reference to local temporary object [-Wreturn-stack-address]
            return "STObject";
```

Reply:
I was able to build this commit without any errors/warnings: 3e5617064ca5c0bac37b3aa8f5316d659fdd69af

Reply:
Oops, I was changing the return type instead of the param type. Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/c34bcd0a628f0c866b0b5bcef0aa5af1e22d43aa